docs/releases/5.2.md

```diff
@@ -98,6 +98,49 @@ depth: 1
 
 Wagtail now uses the Willow feature of providing the image MIME type, which uses the [`filetype`](https://pypi.org/project/filetype/) package.
 
+### Adoption of `classname` convention for `MenuItem` related classes and hooks
+
+Wagtail `MenuItem` and menu hooks have been updated to use the more consistent naming of `classname` (singular) instead of `classnames` (plural), an adoption that started in Wagtail 4.2.
+
+The current keyword argument naming will be supported but will trigger a deprecation warning, support for this variant will be removed in a future release.
+
+The following classes will adopt this new convention.
+
+-   `admin.menu.MenuItem`
+-   `admin.ui.sidebar.ActionMenuItem`
+-   `admin.ui.sidebar.LinkMenuItem`
+-   `admin.ui.sidebar.PageExplorerMenuItem`
+-   `contrib.settings.registry.SettingMenuItem`
+
+The following hooks usage may be impacted if `classnames` were used when generating menu items.
+
+-   `register_admin_menu_item`
+-   `register_admin_menu_item`
```

Comment:
did you mean

```suggestion
-   `register_settings_menu_item`
```

?

Reply:
Yep. Looks like I copy/pasted this and didn't clean it up.